aether-entity-extraction-module/entrypoint.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+#
+# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+set -Eeuo pipefail
+
+function show_help {
+    echo """
+    Commands
+    ----------------------------------------------------------------------------
+    bash              : run bash
+    eval              : eval shell command
+
+    flake8            : check against code style guidelines
+    pip_freeze        : freeze pip dependencies and write to requirements.txt
+
+    start             : start extractor with settings from production
+
+    test              : run unit and integration tests.
+    test_lint         : run flake8 tests
+    test_integration  : run integration tests
+    test_unit         : run unit tests
+    """
+}
+
+function test_flake8 {
+    flake8 /code/. --config=/code/setup.cfg
+}
+
+function after_test {
+    cat /code/conf/extras/good_job.txt
+    rm -R .pytest_cache
+    rm -rf tests/__pycache__
+}
+
+function test_unit {
+    pytest -m unit
+    after_test
+}
+
+function test_integration {
+    pytest -m integration
+    after_test
+}
+
+function test_all {
+    pytest
+    after_test
+}
+
+export APP_MODULE=extractor
+export DJANGO_SETTINGS_MODULE="${APP_MODULE}.settings"
```

Comment:
Do you need these?

Reply:
Yes I do aether-entity-extraction-module/extractor/settings.py

Reply:
Yes I do ==> [Settings](https://github.com/eHealthAfrica/aether/pull/649/files/02895201e0f4a74d4485510e92660ab60cd53c88#diff-4b73dab268e37dbad83df071901c8e7c)